source/common/network/address_impl.cc

```diff
@@ -93,6 +93,7 @@ Address::InstanceConstSharedPtr addressFromSockAddr(const sockaddr_storage& ss,
 
 Ipv4Instance::Ipv4Instance(const sockaddr_in* address, absl::string_view sock_interface)
```

Comment:
In long term I think we want to remove this interface and take explicit `uint32_t address` (or a absl::FixedArray)  and `uint16_t port`. (same for IPv6). leaking sockaddr_in here has been source of use-of-uninitialized-value.